cmd/log_index.cpp

```diff
@@ -0,0 +1,240 @@
+/*
+   Copyright 2021 The Silkworm Authors
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+#include <iomanip>
+#include <iostream>
+#include <string>
+#include <unordered_map>
+
+#include <CLI/CLI.hpp>
+#include <boost/endian/conversion.hpp>
+#include <boost/filesystem.hpp>
+#include <thread>
```

Comment:
Please format this file using our `.clang-format` style. Which IDE do you use? VS Code, for example, picks up `.clang-format` automatically when you Format Document.

Reply:
I am using Vs-Code